pino.js

```diff
@@ -338,6 +343,15 @@ function pino (opts, stream) {
     }
   }
 
+  const baseLog = typeof iopts.baseLog === 'object' ? iopts.baseLog : defaultBaseLog
```

Comment:
* also need to think about processing keys which are objects or arrays: e.g. `{base: { trace: ['a', 'b', 'c']}`